europe/README.EUROPE

```diff
@@ -294,3 +294,111 @@ Residuals at 130 further test points average 0.06m, maximum 0.16m.
 Recommended to transform from Portuguese Datum 73 (EPSG:27493) to ETRS89 in Portugal (EPSG:3763).
 
 * D73_ETRS89_geo.gsb
+
+### Iceland: ISN93 -> ISN2016
+
+2D geodetic transformation from Iceland Datum ISN93 to Iceland Datum ISN2016
+
+*Source*: [National Land Survey of Iceland](https://atlas.lmi.is/LmiData/index.php?id=626468364600)
```

Comment:
For proper display in github, could you make sure that that line (and others starting with '*', except the last one in a series, like "*Credit*" here) are terminated by 2 space character.

Reply:
Updated the file regarding this